### ISSUE TITLE
Postman Shows Password in Basic Auth as plain text

### ISSUE THREAD

**Issue by dbnex14** [a85/POSTMan-Chrome-Extension#12441]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Before when I create a request with Basic Auth, the password field used to be masked and I could click a toggle to view it.

Now, the password field is plain text with a yellow triangle suggesting to use variables for sensitive information.  I do not want to use variables for unimportant reasons.  I just want to be able to enter password and know that it is not visible to someone I am showing my request.  And if I want to view the password, I can toggle to see it and toggle back to mask it again.  A common practice all over the net.

Version
10.19.10
UI version
10.19.10-ui-231026-1128

Desktop platform version
10.19.7

Architecture
x64

OS platform
win32 10.0.19045

### Steps To Reproduce

1. create new GET request
2. in Auth, select Basic Auth as Type
3. enter username
4. enter password < this is plain text but should be masked with a toggle allowing to view password

### Screenshots or Videos

![image](https://github.com/postmanlabs/postman-app-support/assets/8755679/6b2b3685-2f0a-4989-ae4d-a91b5dbea891)

### Operating System

Windows

### Postman Version

10.19.10

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by devr77** [comment]
@dbnex14 Can You assign to me

**Comment by devr77** [comment]
@DannyDainton @numaanashraf Can You assign to me, I Want to work on this

**Comment by DDSAdeo** [comment]
It was not clear for me too, but you must use environment variable with secret type
![image](https://github.com/postmanlabs/postman-app-support/assets/127946621/dcde2452-e1cb-4e7c-a87e-7599b0b34753)

**Comment by dbnex14** [comment]
> It was not clear for me too, but you must use environment variable with secret type ![image](https://private-user-images.githubusercontent.com/127946621/293929210-dcde2452-e1cb-4e7c-a87e-7599b0b34753.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQzMTc4NDgsIm5iZiI6MTcwNDMxNzU0OCwicGF0aCI6Ii8xMjc5NDY2MjEvMjkzOTI5MjEwLWRjZGUyNDUyLWUxY2ItNGU3Yy1hODdlLTc1OTliMGIzNDc1My5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwM1QyMTMyMjhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xOThiZWRmZDYxNTllOTdlMmY5OGQzMDk5NzFkM2QzMDA1MjA3OTQzMDAxZTBhMjFmOGNiNzUxMzRhOGU3MjRkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.p2_He5aqQlbGM6Z8ePMULp54tXGG1W9NHxQpLENBr3M)

Sure but I dont want to use environments.  The password should not be displayed regardless of whether we use environments or not.  Imagine yourself presenting to group of people and need to create a quick request to demo something.  I dont want to waste time setting environments for that.

**Comment by EricChen1248** [comment]
Right, especially for one offs, or to test if a credential is working.
For example, we have an environment variable setup with service account access. However, it's showing a 401 error. I want to quickly substitute it with an admin credentials that has higher access to test whether the issue is actually access level problems, or something else.

I do not want to put the credentials into the Environments, as that is just plain insecure, having admin credentials lying around on disk is poor security. We have dedicated software like vault and password managers that do that, and also handle credential rotations, so postman vault is not a good substitute. 

I can no longer paste in the credentials while screen sharing, or preventing someone from looking at it over my shoulder.

**Comment by dbnex14** [comment]
Yup, absolutely right.  Makes me wonder how come postman team came up with the worst possible idea to shore password entered in the password field totally unmasked.   I haven't seen something so stupid in the industry for at least 20 years.  That is a serious loss of any confidence in the product

> Right, especially for one offs, or to test if a credential is working. For example, we have an environment variable setup with service account access. However, it's showing a 401 error. I want to quickly substitute it with an admin credentials that has higher access to test whether the issue is actually access level problems, or something else.
> 
> I do not want to put the credentials into the Environments, as that is just plain insecure, having admin credentials lying around on disk is poor security. We have dedicated software like vault and password managers that do that, and also handle credential rotations, so postman vault is not a good substitute.
> 
> I can no longer paste in the credentials while screen sharing, or preventing someone from looking at it over my shoulder.
